addons/account/models/account_journal_dashboard.py

```diff
@@ -536,7 +536,7 @@ def open_action_with_context(self):
 
     def create_bank_statement(self):
         """return action to create a bank statements. This button should be called only on journals with type =='bank'"""
-        action = self.env.ref('account.action_bank_statement_tree').sudo().read()[0]
+        action = self.env["ir.actions.actions"]._for_xml_id("account.action_bank_statement_tree")
```

Comment:
`open_action` from https://github.com/odoo/odoo/pull/53335/commits/9db239c5c8985d4d23aede23e47d656aa8292a98 should also be convertible to `_for_xml_id` no?